api/specs/web-server/_computations.py

```diff
@@ -28,6 +28,25 @@ async def get_computation(project_id: ProjectID):
 @router.post(
     "/computations/{project_id}:start",
     response_model=Envelope[_ComputationStarted],
+    responses={
+        status.HTTP_404_NOT_FOUND: {
+            "description": "Project/wallet/pricing details not found"
+        },
+        status.HTTP_402_PAYMENT_REQUIRED: {
+            "description": "Insufficient osparc credits"
+        },
+        status.HTTP_406_NOT_ACCEPTABLE: {
+            "description": "Cluster not found",
+        },
+        status.HTTP_503_SERVICE_UNAVAILABLE: {
+            "description": "Service not available",
```

Comment:
what service? webserver? something else?

Reply:
This is tricky because this one could also come from `director-v2`. So I don't think it is possible to give a general description here. I think name of the service which failed should be in the error message which is sent around instead.

Reply:
ok